src/services/converters/ImageMediaConverter.ts

```diff
@@ -3,27 +3,16 @@ import MediaConverter, { MediaFile } from "./MediaConverter";
 const MAX_IMAGE_DIM = 1024;
 
 export default class ImageMediaConverter extends MediaConverter {
-  checkImage(file: Blob): Promise<boolean> {
-    return new Promise<boolean>((resolve) => {
-      createImageBitmap(file)
-        .then((image: ImageBitmap) => {
-          if (image.width > MAX_IMAGE_DIM || image.height > MAX_IMAGE_DIM) {
-            resolve(false);
-          } else {
-            resolve(true);
-          }
-        })
-        .catch(() => {
-          resolve(false);
-        });
-    });
+  async checkImage(file: Blob): Promise<boolean> {
+    const image = await createImageBitmap(file);
```

Comment:
Can this throw some kind of exception? For example, if the file isn't an image, I'd imagine it throws *something* or returns `null`, but in the latter would be problematic.

Reply:
`createImageBitmap` itself throws an error, but I wrapped it up in my own